tests/benchmark/benchmark_apis_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package benchmark
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/rpc"
+	"google.golang.org/api/iterator"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+func apiId(i int) string {
+	return fmt.Sprintf("test-%d", i)
+}
+
+func apiName(apiId string) string {
+	return fmt.Sprintf("%s/apis/%s", root(), apiId)
+}
+
+func getApi(b *testing.B, ctx context.Context, client connection.Client, apiId string) error {
+	b.Helper()
```

Comment:
Since this function doesn't actually fail/error/log using `b`, I don't think we gain anything from marking it as a helper (and can drop it entirely from the function).